console/lunox.ts

```diff
@@ -31,7 +33,7 @@ program
       await buildClient();
       console.log(green("view are compiled to ./dist/client folder\n"));
       console.log(blueBright("copying assets..."));
-      await runCommand("cp -r ./public ./dist/public");
+      copyHelper(base_path("public"), base_path("dist/public"));
```

Comment:
we cannot use `base_path` and `lunox_path` helper here. I will explain it later. So just use `path.join(process.cwd(), "public")` and  `path.join(process.cwd(), "dist/public")`

Reply:
`base_path` is configured on lunox skeleton, see https://github.com/kodepandai/lunox/blob/eaba68fa1462eaf0f1590ceb6b007c29d3de92b9/bootstrap/app.ts#L6-L7
Can you see that base_path helper is refere to `dist` folder on lunox skeleton (after we run dev or prod)

`lunox_path` is refer to `dist` folder. So when we call this function on lunox skeleton, `lunox_path` will resolved to
`node_modules/lunox/dist`. Think that this is helper to resolve lunox framework root on node_modules folder

Because of this, we should call lunox_path or base_path in main application, not in builder phase